From 098edb5f70a151a3f377478fd6e0d08ed56b8ef7 Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Fri, 11 Mar 2022 00:21:53 +0100
Subject: [PATCH] build: Make installation directory overridable

This will make it easier for Linux distros using non-standard paths to install.
---
 Makefile | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 779a618..7948866 100644
--- a/Makefile
+++ b/Makefile
@@ -18,8 +18,7 @@ makewin:
 	gcc -mwindows -o ./bin/win32/bimp -O2 $(CFLAGS) src/*.c src/manipulation-gui/*.c src/images/*.c $(LDFLAGS)
 		
 install: 
-	mkdir -p "$(USER_INSTALL_DIR)"
-	gimptool-2.0 --install-bin ./bin/bimp
+	install -Dt "$(USER_INSTALL_DIR)" ./bin/bimp
 	cp -Rf ./bimp-locale/ "$(USER_INSTALL_DIR)"
 	
 uninstall: 
@@ -27,8 +26,7 @@ uninstall:
 	rm -R $(USER_INSTALL_DIR)/bimp-locale
 
 install-admin:
-	mkdir -p "$(SYSTEM_INSTALL_DIR)"
-	gimptool-2.0 --install-admin-bin ./bin/bimp
+	install -Dt "$(SYSTEM_INSTALL_DIR)" ./bin/bimp
 	cp -Rf ./bimp-locale/ "$(SYSTEM_INSTALL_DIR)"
 
 uninstall-admin:
